sdk/python/lib/pulumi/resource.py

```diff
@@ -159,12 +161,13 @@ class Alias:
     The previous project of the resource. If not provided, defaults to `pulumi.getProject()`.
     """
 
+    #ignoring ... types as this value is used as a sentinal marker
```

Comment:
sorry, i don't understnad what this comment means can you clarify (either here on in hte code)?

Reply:
This one was your doing @CyrusNajmabadi  :-).  You played this trick of using `...` as a sentinel value different from `None`.  But that's an implementation detail, and not part of the desired user-visible type of the API.  So these ignores ensure we don't have to union `...` into the user-visibile type.

Reply:
ah.  'click'.  thanks.

Reply:
Yup, unclear after a second read. Updated the comment.